manifests/params.pp

```diff
@@ -358,6 +438,7 @@
       $client_package_name      = 'snmp'
       $client_config            = '/etc/snmp/snmp.conf'
 
+      $trap_service_name        = 'snmptrapd'
```

Comment:
In `init.pp`, `$trap_service_name` is defaulted to `$snmp::params::trap_service_name`, so if that is happening on Debian, the default will not exist in params, which can lead to warnings in P4 about undefined variables.